src/reactor.js

```diff
@@ -26,9 +27,13 @@ class Reactor {
   constructor(config = {}) {
     const debug = !!config.debug
     const baseOptions = debug ? DEBUG_OPTIONS : PROD_OPTIONS
+    // if defined, merge the custom implementation over the noop logger to avoid undefined lookups,
+    // otherwise, just use the built-in console group logger
+    const debugLogger = config.logging ? extend({}, NoopLogger, config.logging) : ConsoleGroupLogger
```

Comment:
Since we are passing `Loggers` here i think the config option should be called `Logger`